react/src/pages/admin/UserRow.tsx

```diff
@@ -1,103 +1,108 @@
-                    margin="dense"
-                    type="password"
-                    autoComplete="new-password"
-                    label="Confirm password"
-                    value={confirmPassword}
-                    onChange={e => setConfirmPassword(e.target.value)}
-                />
-            </CardContent>
-            <CardActions disableSpacing className={classes.actions}>
-                <Button variant="outlined" color="secondary" onClick={() => props.onDelete(state)}>
-                    <DeleteOutlineIcon />
-                    Delete
-                </Button>
-                <Button
-                    variant="outlined"
-                    color="primary"
-                    className={classes.expand}
-                    onClick={() => props.onUpdate(state)}
-                >
-                    <EditIcon />
-                    Update
-                </Button>
-            </CardActions>
-        </Card>
+        <Grid item sm={12} md={6}>
+            <Paper>
+                <ListItem>
+                    <ListItemAvatar className={classes.icon}>
+                        {!props.user.deactivated ? (
+                            <Person fontSize="large" />
+                        ) : (
+                            <PersonOutline fontSize="large" />
+                        )}
+                    </ListItemAvatar>
+                    <ListItemText
+                        disableTypography
+                        primary={
+                            <Grid container {...gridProps}>
+                                <Grid item>
+                                    <Typography variant="h6" className={classes.title}>
+                                        {props.user.username}{" "}
+                                        {props.user.isAdmin && <Security fontSize="inherit" />}
+                                    </Typography>
+                                </Grid>
+                                <Grid item>
+                                    <LastLoginDisplay date={date} time={time} />
+                                </Grid>
+                            </Grid>
+                        }
+                        secondary={
+                            <Grid container {...gridProps}>
+                                <Grid item>
+                                    <Typography variant="subtitle1" className={classes.title}>
+                                        {props.user.email}
+                                    </Typography>
+                                </Grid>
+                                <Grid item>
+                                    <ChipGroup names={["CHEO", "SK"]} size="small" />
+                                </Grid>
+                            </Grid>
+                        }
+                    />
+                    <IconButton onClick={e => setOpen(!open)} className={classes.button}>
+                        {open ? <ExpandMore /> : <ExpandLess />}
```

Comment:
To be consistent with other lists, this should be
```suggestion
                        {open ? <ExpandLess /> : <ExpandMore />}
```